### PR TITLE
re enable cluster tests

### PR DESCRIPTION
**Pull request by davejrt**
Successful main-dry-run [here](https://buildkite.com/sourcegraph/sourcegraph/builds/120951#94ee3432-828c-4a71-8844-c6a04babd1e9)

part of https://github.com/sourcegraph/sourcegraph/issues/25977

### CODE REVIEW

**Comment by bobheadxi** [comment]
Linking for reference: https://github.com/sourcegraph/sourcegraph/issues/25977

Also closes https://github.com/sourcegraph/sourcegraph/pull/25748

File: dev/ci/test/cluster/cluster-test.sh [hunk 1]
```diff
@@ -44,26 +44,23 @@ function cluster_setup() {
...
   while IFS= read -r line; do
     echo "$line"
-    grep -lr '.' -e "index.docker.io/sourcegraph/$line" --include \*.yaml | xargs sed -i -E "s#index.docker.io/sourcegraph/$line:.*#us.gcr.io/sourcegraph-dev/$line:$CANDIDATE_VERSION#g"
+    grep -lr './base/' -e "index.docker.io/sourcegraph/$line" --include \*.yaml | xargs sed -i -E "s#index.docker.io/sourcegraph/$line:.*#us.gcr.io/sourcegraph-dev/$line:$CANDIDATE_VERSION#g"
   done < <(printf '%s\n' "$DOCKER_CLUSTER_IMAGES_TXT")
-  popd
+
   echo "--- create cluster"
-  ./create-new-cluster.sh
+  ./overlay-generate-cluster.sh low-resource generated-cluster
```
Comment (bobheadxi) [inline dev/ci/test/cluster/cluster-test.sh:58]: Does this already have cadvisor removed?

Comment (davejrt) [reply dev/ci/test/cluster/cluster-test.sh:58]: It does :-) 

https://github.com/sourcegraph/deploy-sourcegraph/commit/a3e354cb4542794683c6df1ee154972b6f08543a

Comment (bobheadxi) [reply dev/ci/test/cluster/cluster-test.sh:58]: nice! :-)

**Review by bobheadxi — APPROVED** [review]
woo

**Comment by bobheadxi** [comment]
@davejrt should we remove this auto-pass? https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/dev/ci/test/cluster/cluster-test.sh?L119

**Comment by davejrt** [comment]
> @davejrt should we remove this auto-pass? https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/dev/ci/test/cluster/cluster-test.sh?L119

Sure, let me fix that quickly

**Pull request merged**